ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -4082,7 +4082,9 @@ let vm_migrate printer rpc session_id params =
       let open Xmlrpc_client in
       let http = xmlrpc ~version:"1.0" "/" in
       XMLRPC_protocol.rpc ~srcstr:"cli" ~dststr:"dst_xapi"
-        ~transport:(SSL (SSL.make ~use_fork_exec_helper:false (), ip, 443))
+        ~transport:
+          (SSL
+             (SSL.make ~verify_cert:None ~use_fork_exec_helper:false (), ip, 443))
```

Comment:
This one is a tough one. It's' cross-pool migration started from the cli. If trust is to be established beforehand the cli needs to be changed to have the certificate of the remote host as trusted while using the SNI 'pool'.